aodndata/soop/soop_co2_nrt_nc_generator.py

```diff
@@ -234,27 +221,22 @@ def check_parameters(dataf, vessel_code, input_param, src_file):
     Checks that Lat/Lon are not all missing.
     Returns updated dataframe
     """
-    rt_input_parameters = set.union(INPUT_RT_PARAMETERS,
-                                    eval(vessel_code + '_SPECIFIC_INPUT_PARAMS'))
-    if not all(param in input_param for param in rt_input_parameters):
-        missing_param = []
-        for required_param in rt_input_parameters:
+    missing_param = []
+    if not all(param in input_param for param in INPUT_RT_PARAMETERS):
+        for required_param in INPUT_RT_PARAMETERS:
             if required_param not in input_param:
-                missing_param = missing_param.append(required_param)
-                raise InvalidFileContentError(
-                    "Missing parameter(s) '{missing_param}' in file '{src_file}'.Aborting".format(
-                        missing_param=missing_param, src_file=src_file))
+                missing_param.append(required_param)
     else:  # required_param all present . Change dtype to numeric where relevant
```

Comment:
is that okay to remove the `raise InvalidFileContentError` if there are missing param?

Reply:
I wanted to raise a warning instead but forgot. But you're right, best to keep it,i've  put the  `InvalidFileContentError` back in